Demo/Common/IntegrationExamples/DropIn.swift

```diff
@@ -28,13 +28,16 @@ internal final class DropIn: DropInExampleProtocol {
     // MARK: - Networking
 
     internal func requestInitialData() {
-        requestAdyenSessionConfiguration { adyenSessionConfig, errorResponse in
+        requestAdyenSessionConfiguration { [weak self] adyenSessionConfig, errorResponse in
+            guard let strongSelf = self else {
```

Comment:
no need to call it strongSelf anymore, `let self = self else` works

Reply:
Changed here  a840aa9218d8d736818e3e1e33b478ad6b50de8d